zerver/lib/actions.py

```diff
@@ -3395,13 +3395,13 @@ def user_email_is_unique(email):
         pass
 
 def do_invite_users(user_profile, invitee_emails, streams):
-    # type: (UserProfile, SizedTextIterable, Iterable[Stream]) -> Tuple[Optional[str], Dict[str, List[Tuple[text_type, str]]]]
+    # type: (UserProfile, SizedTextIterable, Iterable[Stream]) -> Tuple[Optional[str], Dict[str, Any]]
```

Comment:
Uh, my bad, didn't see the other comment.
@TomaszKolek Since the client needs to know if it should remove emails that were valid or no. (if no notifications were actually sent, all emails should be kept for resubmitting).